_posts/jwarrich/2016-01-26-Javairias-treasure-hunt.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+author: jwarrich
+title: "Javairia's Treasure Hunt"
+---
+
+Here is the treasure hunt:
+
+<iframe src="https://trinket.io/embed/python/abb78e20b8" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+When doing this assignment, I started off a little confused. I had to go back and look at what loops are and how they work. Once I understood that, I looked to see what all I needed to accomplish and took it step by step. One thing that I got stuck on was creating the ranges and knowing whether the turtle color was working according to the range. Because I didn't know what the random x and y coordinates were, it was hard to know whether I was doing the assignment correctly.
```

Comment:
Good reflection!  Sounds like you were able to overcome quite a few challenges.

One tip for next time: you could consider printing out things like `treasure_x` and `treasure_y` while you're developing, then removing those print statements when you're done.  This is a good technique for everyone to know- I'll mention it in class or maybe in a howto.